src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -290,11 +290,15 @@ export default {
 			updateSearchState(this.apollo, filters, this.allFacets, this.queryType, this.loanSearchState);
 		},
 		handleUpdatedFilters(filters) {
-			this.updateState({ ...this.loanSearchState, ...filters });
+			this.updateState({ ...this.loanSearchState, ...filters, pageOffset: 0 });
```

Comment:
Is this always 0 or should it be set based on current state?

Reply:
The `handleUpdatedFilters` is called when a filter/sorting changes, so it resets to the first page. This aligns with the previous behavior and ensures the user never has a page beyond the newly filtered results. Alternatively we could ensure the page is only changed to the last page available when filters change and shrink the results, but I noticed that we used to reset, and that's a simpler approach.

Reply:
Ok cool, that makes sense!